docs/admin.rst

```diff
@@ -3,47 +3,52 @@ Admin Guide
 
 .. include:: ./includes/provide-feedback.txt
 
-You (the admin) should have your own username and passphrase, plus
-two-factor authentication through either the FreeOTP app
-on your smartphone or a YubiKey.
+The SecureDrop architecture contains multiple machines and hardened servers.
+While we have automated many of the installation and maintenance tasks, a
+skilled Linux admin is required to responsibly run the system.
+
+This section outlines the tasks the admin is responsible for in order to
+ensure that the SecureDrop server continues to be a safe place for sources to
+talk to journalists.
 
 .. _Responsibilities:
 
 Responsibilities
 ----------------
 
-The SecureDrop architecture contains multiple hardened servers, and while we have
-automated many of the installation and maintenance tasks, a skilled Linux
-admin and some manual intervention is required to responsibly run the system.
+Maintaining Credentials
+~~~~~~~~~~~~~~~~~~~~~~~
 
-This section outlines the tasks the admin is responsible for in order to
-ensure that the SecureDrop server continues to be a safe place for sources to
-talk to journalists.
+The admin should have her own username, passphrase, and two-factor
+authentication method (via smartphone app or YubiKey). Admins are also
+responsible for tracking user credentials and encouraging best practices. (See
```

Comment:
What does it mean by "tracking user credentials"? (I thought for a second it might be interpreted as "keep a record of all user's passwords", which we don't want)

Reply:
managing?